Makefile

```diff
@@ -5,7 +5,7 @@ NATIVEARCH	 := $(shell go version | awk -F '[ /]' '{print $$5}')
 TOOLS_DIR    := ./bin/dev-tools
 INTEGRATION  = nri-kube-events
 GOLANGCILINT_VERSION = 1.17.1
```

Comment:
On https://github.com/newrelic/k8s-metadata-injection/pull/73/ I also have an outdated version of golangci (1.12), perhaps we could use this opportunity to update both of them. What do you think?

Reply:
Update: I upgraded the PR above to use golangci lint v1.32 just to avoid missing the chance as we don't know where these will be touched again :)

Reply:
I agree!

Reply:
1.33.0 just 2 days ago :)